apps/lfx-one/src/app/modules/newsletters/components/newsletter-preview/newsletter-preview.component.scss

```
@@ -0,0 +1,198 @@
+  }
+
+  &__body {
+    padding: 1.5rem 2rem 2rem;
+    font-size: 1rem;
+    color: theme('colors.gray.800');
+    line-height: 1.65;
+
+    // Content is injected via [innerHTML] so it lives outside this component's
+    // view encapsulation. ::ng-deep is required for these rules to reach those
+    // elements. Also restores list-style which Tailwind preflight strips.
+    ::ng-deep {
```

Stylelint (17.11.1)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)